dev-qt/qtprotobuf/qtprotobuf-0.1.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="gRPC and Protobuf generator and bindings for Qt framework"
+HOMEPAGE="https://semlanik.github.io/qtprotobuf/"
+SRC_URI="https://github.com/semlanik/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-lang/go-1.10
+	>=dev-libs/protobuf-3.6:=
+	>=dev-qt/qtcore-5.12.3
+	>=dev-qt/qtnetwork-5.12.3
+	>=dev-qt/qtdeclarative-5.12.3"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=OFF
```

Comment:
why that?

Reply:
There is static linkage of internal libraries. If this flag is enabled, unexported static libraries targets error occured at configuration phase.

Reply:
that would be something to fix for upstream's cmake.

Reply:
It's part of library design so far. In 0.2.0 release I will make this flag disabled localy